content/en/docs/concepts/configuration/manage-compute-resources-container.md

```diff
@@ -27,6 +27,8 @@ the difference between requests and limits, see
 
 *CPU* and *memory* are each a *resource type*. A resource type has a base unit.
 CPU is specified in units of cores, and memory is specified in units of bytes.
+Starting from version 1.16, *Hugepages* also can be specified. Typical sizes are 2Mi and 1Gi.
```

Comment:
I'm not sure what this is saying to me.

How about:
```markdown
If you're using Kubernetes v1.14 or newer, you can specify _huge page_ resources.
Huge pages are a Linux-specific feature where the node kernel allocates blocks of memory
that are much larger than the default page size.

For example, on a system where the default page size is 4KiB, you could specify a limit,
`hugepages-2Mi: 80Mi`. If the container tries allocating over 40 2MiB huge pages (a
total of 80 MiB) in total, that allocation fails.
```

(according to https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/ HugePages went GA in v1.14 - if that's wrong, there more fixing needed).

Reply:
The suggested changes make more sense @sftim